src/jit/lsra.cpp

```diff
@@ -2607,44 +2607,39 @@ void LinearScan::checkLastUses(BasicBlock* block)
 
             if (currentRefPosition->refType == RefTypeDef || currentRefPosition->refType == RefTypeDummyDef)
             {
-                VarSetOps::RemoveElemD(compiler, temp, varIndex);
+                VarSetOps::RemoveElemD(compiler, lastUses, varIndex);
             }
         }
 
         assert(currentRefPosition != refPositions.rend());
         ++currentRefPosition;
     }
 
-    VARSET_TP temp2(VarSetOps::MakeCopy(compiler, block->bbLiveIn));
-    VarSetOps::DiffD(compiler, temp2, temp);
-    VarSetOps::DiffD(compiler, temp, block->bbLiveIn);
+    VARSET_TP usedAsDead(VarSetOps::Diff(compiler, block->bbLiveIn, lastUses));
```

Comment:
These names are more confusing than enlightening to me. Perhaps `usedAsDead` could be better named as `liveInNotComputedLive` and `liveNotUsed` as `complutedLiveNotLiveIn`.